tokens/BSC.json

```diff
@@ -94,5 +94,13 @@
     "decimals": 18,
     "chainId": 56,
     "logoURI": "https://assets.coingecko.com/coins/images/17816/small/relay-logo-200.png?1629339288"
+  },
+  {
+    "name": "Brazilian Digital",
+    "address": "0x71be881e9C5d4465B3FfF61e89c6f3651E69B5bb",
+    "symbol": "BRZ",
+    "decimals": 18,
```

Comment:
This token only has 4 decimals. You can check this in the explorer: https://bscscan.com/token/0x71be881e9C5d4465B3FfF61e89c6f3651E69B5bb

Note that the decimals may differ between chains, please always check on all chains.